lib/modules/manager/bundler/artifacts.ts

```diff
@@ -28,6 +32,23 @@ import {
 
 const hostConfigVariablePrefix = 'BUNDLE_';
 
+export function buildArgs(config: UpdateArtifactsConfig): string[] {
+  // --major is the default and does not need to be handled separately.
+  const patch = config.updateType === 'patch' && '--patch';
+  const minor = config.updateType === 'minor' && '--minor';
+
+  // --patch and --minor flags are only suggestions, use --strict to enforce them.
+  const strict = (patch || minor) && '--strict';
+
+  const conservative =
+    config.postUpdateOptions?.includes('bundlerConservative') &&
+    '--conservative';
+  const args = [patch, minor, strict, conservative, '--update'].filter(
+    is.nonEmptyString
+  );
+  return args;
+}
```

Comment:
```suggestion
  const args: string[] = [];
  // --major is the default and does not need to be handled separately.
  switch(config.updateType) {
    case 'patch':
      args.push('--patch',  '--strict');
      break;
    case 'minor':
      args.push('--minor', '--strict');
      break;
  }

  if (config.postUpdateOptions?.includes('bundlerConservative')) {
    args.push('--conservative');
  }

  args.push('--update');
  return args;
}
```

this is much cleaner 😉

Reply:
I don't like mutating stuff, but on the other hand, this prevents us from using the `is.nonEmptyString` filtering, so let's do it this way 👍